static/elements/chromedash-process-overview.js

```diff
@@ -229,6 +255,15 @@ class ChromedashProcessOverview extends LitElement {
       nothing }
 
     </div>
+
+    <chromedash-dialog heading="Progress Steps Missing">
+      <ol class="pending">
+        ${this.pendingItems.map((item) => html`<li>${item}</li>`)}
+      </ol>
+      <a href="/guide/stage/${featureId}/${this.pendingItems.stage ? this.pendingItems.stage.outgoing_stage : ''}"
+        target="_blank" class="button primary">Edit Steps</a>
+      <a href="${this.pendingItems.continueUrl}" target="_blank">Continue to Draft Email</a>
```

Comment:
This button label is kind of long.
How about just: "Continue" or "Proceed"?

Also, this seems like a case where the button should have a border, but not be filled in to look like the primary button.    And, the buttons are crowded too close together.

Try:
margin: var(--content-padding);
border: var(--default-border);
border-color: var(--primary-border-background);
padding: 4px 16px;
border-radius: var(--border-radius);

You could just do that here in this component, or if you are feel like diving into our CSS you could add some "outlined button" variables to theme.css and then define a new button CSS class in shared.scss.